librz/debug/trace.c

```diff
@@ -209,57 +209,34 @@ static int cmpaddr(const void *_a, const void *_b) {
 														     : 0;
 }
 
-RZ_API void rz_debug_trace_list(RzDebug *dbg, int mode, ut64 offset) {
+RZ_API RzList *rz_debug_traces_info(RzDebug *dbg, ut64 offset) {
```

Comment:
Please add Doxygen and `rz_return_val_if_fail(dbg, NULL)`.

Reply:
I also recommend adding `RZ_OWN` for the return result here and in the header file.